lbry/schema/claim.py

```diff
@@ -253,7 +253,7 @@ def update(self, file_path=None, height=None, width=None, duration=None, **kwarg
         if stream_type in ('image', 'video', 'audio'):
             media = getattr(self, stream_type)
             media_args = {'file_metadata': None}
-            if file_path is not None:
+            if file_path is not None and (not duration or not width or not height):
```

Comment:
I think this can be `if file_path is not None and not all((duration, width, height))` but please confirm.

Reply:
it can be that "all" approach. unfortunately t only saves a couple of chars